vscode-dotnet-runtime-library/src/Acquisition/InstallScriptAcquisitionWorker.ts

```diff
@@ -40,7 +46,7 @@ export class InstallScriptAcquisitionWorker implements IInstallScriptAcquisition
             }
 
             Debugging.log('Writing the dotnet install script into a file.');
-            this.writeScriptAsFile(script, this.scriptFilePath);
+            await this.writeScriptAsFile(script, this.scriptFilePath);

```

Comment:
So this code change successfully ensures serialized access to the writing of the file, but we still attempt to write the file on every acquisition request, right? It would be best if we could avoid doing that work unless necessary (script file content changed, file checksum on disk is incorrect, etc).

Reply:
Yep. Fixed 👌